include/aspect/compat.h

```diff
@@ -28,6 +28,353 @@
 #include <functional>
 #include <memory>
 
+#if !DEAL_II_VERSION_GTE(9,2,0)
```

Comment:
as it doesn't exist in deal.II yet (right?), I think we need to use this unconditionally here. Otherwise, you need to have a ``using `` directive in the ``#else`` branch, right?

Reply:
Ok, so I'll remove this for the current PR, and we'll add it again when the changes are adding to deal.II.

Reply:
Yes.